heartbeat/monitors/util.go

```diff
@@ -342,3 +448,55 @@ func filterIPs(ips []net.IP, filt func(net.IP) bool) []net.IP {
 	}
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function MakeJobSetting should have comment or be unexported